ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-release-ocm-2.4.yaml

```diff
@@ -25,8 +25,12 @@ build_root:
     namespace: openshift
     tag: golang-1.16
 images:
-- dockerfile_path: Dockerfile.test-infra
+- dockerfile_literal: |
+    FROM base
+    COPY . .
   from: src
+  to: assisted-test-infra
+- dockerfile_path: Dockerfile.assisted-test-infra
```

Comment:
This path exists on ``release-ocm-2.4`` branch?
If not then we can probably use ``Dockerfile.test-infra``

Reply:
Yes, It's part of https://github.com/openshift/assisted-test-infra/pull/1426